components/cd-hr/README.md

```diff
@@ -0,0 +1,7 @@
+# Separator Component
+
+## Purpose and Usage
+It's a horizontal rule, copied from GHO - an exercise only.
```

Comment:
Probably a good exercise to fill this out - why would we need a horizontal rule? When and how should a content editor use this?